packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -22,12 +17,26 @@ import {
     neutralOutlineHover,
     neutralOutlineRest,
 } from "../utilities/color";
+import { applyCornerRadius } from "../utilities/border";
+import { applyFontSize, densityToTypeOffset, padding } from "../utilities/density";
+import { applyDisabledState } from "../utilities/disabled";
 
 const styles: ComponentStyles<CheckboxClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<CheckboxClassNameContract, DesignSystem> => {
     const designSystem: DesignSystem = withDesignSystemDefaults(config);
     const direction: Direction = designSystem.direction;
+    const size: number =
+        (designSystem.defaultHeightMultiplier + designSystem.density) *
+            (designSystem.designUnit / 2) +
+        designSystem.designUnit;
```

Comment:
This feels a bit arbitrary in order to achieve a certain value(outcome) for radio/checkbox.  Primarily confused by what the principle is here:
```
(designSystem.designUnit / 2) +
        designSystem.designUnit;
```

Reply:
Updated to tie to height better.